js/src/ui/CurrencySymbol/currencySymbol.example.js

```diff
@@ -0,0 +1,51 @@
+// Copyright 2015, 2016 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { Component } from 'react';
+
+import PlaygroundExample from '~/playground/playgroundExample';
+
+import ConnectedCurrencySymbol, { CurrencySymbol } from './currencySymbol';
+
+export default class CurrencySymbolExample extends Component {
+  render () {
+    return (
+      <div>
+        <PlaygroundExample name='Connected Currency Symbol'>
+          <ConnectedCurrencySymbol />
+        </PlaygroundExample>
+
+        <PlaygroundExample name='Simple Currency Symbol'>
+          <CurrencySymbol
```

Comment:
100% agreed on the final purpose - this one is funny since it is just a span. However, think the example could add value going forward, although maybe not quite initially. (Which is the function of the actual component implementation, not the intent of the example)